common-text-blocks.md

```diff
@@ -0,0 +1,31 @@
+
+# Reusable reference blocks
+
+This section is for reusable blocks to establish standards for presentation
+within our Documentation in regards to our blocks of warnings and other
+common areas. These use Vuepress' slots for danger, warning, tip and details.
+Use the first two as sparingly as possible unless ***ABSOLUTELY*** necessary.
+
+
+## For sections that are under construction.
+
+``` ::: warning Under Construction
+
+This document is under construction, please check back soon for updates.
+Please see [our socials](link/to/socials/list) and feel free to ask for assistance or enquire as to the status of this document.
+
+:::
+```
+
+
+## For archived sections
+
+```
+
+::: danger STOP
+
+This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar you use this at your own risk.
```

Comment:
```suggestion
This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar, use this at your own risk.
```